### PR TITLE
fix: inconsistent page layout in docs [Fixes: #8364]

### LINKED ISSUE #8364: Inconsistent padding/margin on docs pages
### Describe the bug

The padding of the opcodes page looks off:
https://ethereum.org/en/developers/docs/evm/opcodes

### To reproduce

1. Go to https://ethereum.org/en/developers/docs/evm/opcodes
2. See margin/padding of main content

### Expected behavior

Should look same as other docs pages.

### Screenshots

Opcodes page:
Image 2022-10-25 at 10 29 18 AM

Normal page:
Image 2022-10-25 at 10 29 35 AM

### Desktop (please complete the following information)

Brave MacOS

### Smartphone (please complete the following information)

_No response_

### Additional context

Perhaps a result of https://github.com/ethereum/ethereum-org-website/pull/8332 ?

### PR DESCRIPTION
**Pull request by amit-ksh**
Made the changes as proposed by @nloureiro in the issue.

1) Removed the `margin: 0 auto` for keeping the content left aligned 
2) Set the `max-width: 16rem` to the table of content
3) Set the `max-width: 756px` to the content container

## Related Issue
Fixes #8364

### CODE REVIEW

**Review by corwintines — CHANGES_REQUESTED** [review]
Hey @amit-ksh. 

Thanks for opening this. I noticed this doesn't quite fit the solution that @nloureiro proposed here: https://github.com/ethereum/ethereum-org-website/issues/8364#issuecomment-1316844297

Let me know if you have any questions :)

**Comment by amit-ksh** [comment]
Hey @corwintines! can you please check this issue also #8373, it might be related

**Comment by nloureiro** [comment]
As far as I could check, this is aligned with the Design system.
I made some comments in the code, but my big question is more about having hardcoded values in the component. Should we have this in a variable? might need to be re-used in other situation in the future and should not be inside a component

**Comment by corwintines** [comment]
@amit-ksh if you have a moment, could you resolve the conflicts here?

**Comment by amit-ksh** [comment]
Hey @corwintines, I try to fix the merge conflict, could please check if I did it correctly or not, first I'm doing this.

File: docs/ds-implementation.md [hunk 1]
```diff
@@ -17,10 +17,10 @@ Follow the new component directory structure:
...
 src/
 └── components/
-    └── ComponentA/
-        ├── index.tsx
-        ├── ComponentA.stories.tsx
-        └── // Any other files as applicable (utils, child components, useHook, etc.)
+└── ComponentA/
+├── index.tsx
+├── ComponentA.stories.tsx
+└── // Any other files as applicable (utils, child components, useHook, etc.)
```
Comment (wackerow) [inline docs/ds-implementation.md:23]:
```suggestion
    └── ComponentA/
        ├── index.tsx
        ├── ComponentA.stories.tsx
        └── // Any other files as applicable (utils, child components, useHook, etc.)
```

**Review by wackerow — COMMENTED** [review]
@amit-ksh sorry for delays! Circled back and was taking a look here... I merged in `dev` and cleared the conflicts...

@nloureiro, if you get a moment take a peak here and let us know if this is still needed...  if so, and if it builds properly, I think we can pull this in.

**Comment by nloureiro** [comment]
I have been reviewing this. Can I propose 2 small changes?

1 . The table of contents still bounces left/right depending on the content. It needs to justify to left on the flex
2. The content is, in fact, 768px but includes the padding. My goal is to have a max-width for the text lines itself. on the dev tools, I simulate something like `max-width: calc( 48rem + var(--eth-space-16) + var(--eth-space-16));` but not sure how the code is set up to solve this in a good way.

I made some notes on these screenshots to help illustrate these 2 points.
Screen Shot 2023-09-22 10 33 19 AM

Screen Shot 2023-09-22 10 33 27 AM


@amit-ksh If I'm not clear on any of these please reach out, Thank you for your help :)

**Comment by amit-ksh** [comment]
@nloureiro @wackerow my university exams are about to start, so, I will watch this after 5th October

**Comment by amit-ksh** [comment]
Hey @nloureiro, here's my solution for the changes you requested: 

1. I believe the `TableOfContent` is not bouncing but the width values are different. It is happening because the width can vary from 12rem(min-width) to 16rem(max-width). If the width of **TableOfContent** should be the same for all the pages then, we can do that by setting the width to some fixed value. 
2. And for setting the max-width just for the content we can use this `box-sizing: content-box`. This will not include the padding and applies the width size to content only.

**Comment by wackerow** [comment]
Hey @amit-ksh, appreciate the efforts here!... recently have merged a fix for this in #12460 so going to close this out. Apologies we weren't able to get this in 🙏

**Pull request closed without merging**